src/Microsoft.Graph.Core/Serialization/Serializer.cs

```diff
@@ -4,9 +4,11 @@
 
 namespace Microsoft.Graph
 {
+    using System;
```

Comment:
Where are the newly added using being used?

Reply:
Thank you. That is the leftover of thrown away code. Will push new in a couple